src/panels/config/helpers/dialog-helper-detail.ts

```diff
@@ -133,45 +133,54 @@ export class DialogHelperDetail extends LitElement {
       items.sort((a, b) => a[1].localeCompare(b[1]));
 
       content = html`
-        ${items.map(([domain, label]) => {
-          // Only OG helpers need to be loaded prior adding one
-          const isLoaded =
-            !(domain in HELPERS) || isComponentLoaded(this.hass, domain);
-          return html`
-            <mwc-list-item
-              .disabled=${!isLoaded}
-              .domain=${domain}
-              @click=${this._domainPicked}
-              @keydown=${this._handleEnter}
-              dialogInitialFocus
-              graphic="icon"
-            >
-              <img
-                slot="graphic"
-                loading="lazy"
-                src=${brandsUrl({
-                  domain,
-                  type: "icon",
-                  useFallback: true,
-                  darkOptimized: this.hass.themes?.darkMode,
-                })}
-                referrerpolicy="no-referrer"
-              />
-              <span class="item-text"> ${label} </span>
-            </mwc-list-item>
-            ${!isLoaded
-              ? html`
-                  <paper-tooltip animation-delay="0"
-                    >${this.hass.localize(
-                      "ui.dialogs.helper_settings.platform_not_loaded",
-                      "platform",
-                      domain
-                    )}</paper-tooltip
-                  >
-                `
-              : ""}
-          `;
-        })}
+        <mwc-list
+          innerRole="listbox"
+          itemRoles="option"
+          innerAriaLabel=${this.hass.localize(
+            "ui.panel.config.helpers.dialog.create_helper"
+          )}
+          rootTabbable
+          dialogInitialFocus
+        >
+          ${items.map(([domain, label]) => {
+            // Only OG helpers need to be loaded prior adding one
+            const isLoaded =
+              !(domain in HELPERS) || isComponentLoaded(this.hass, domain);
+            return html`
+              <mwc-list-item
+                .disabled=${!isLoaded}
+                .domain=${domain}
+                @request-selected=${this._domainPicked}
```

Comment:
Can we switch to the `selected` event of `mwc-list` now?

Reply:
I would use the `action` event, but yes.  You'd have to either save `items` to the class or move it to a getter outside `render` though in order to use the selected index in the handler.  This was just less work 💨.